src/devtools/client/inspector/rules/actions/rules.ts

```diff
@@ -56,10 +56,10 @@ export function updateHighlightedSelector(
  * @param  {Array} rules
  *         Array of Rule objects containing the selected element's CSS rules.
  */
-export function updateRules(rules: Rule[]): UpdateRulesAction {
+export function updateRules(rules: Rule[] | null): UpdateRulesAction {
   return {
     type: "UPDATE_RULES",
-    rules: rules.map(rule => getRuleState(rule)),
+    rules: rules?.map(rule => getRuleState(rule)),
```

Comment:
perhaps here we should do  `rules ? rules.map(rule => getRuleState(rule)) : []`

Reply:
No, see my comment above.

Reply:
got it!